pages/index.tsx

```diff
@@ -40,7 +40,7 @@ export default function Home() {
         errorFound(e);
       }
     }
-    fetchGifts().catch((e) => console.error(e));
+    void fetchGifts();
```

Comment:
`void fetchGifts();` void-lauseke lisätty fetchGifts():in eteen. ESLint vaati tuolle jonkin käsittelytavan (then, catch tai void)